app/services/analytics_events.rb

```diff
@@ -1541,6 +1544,44 @@ def saml_auth_request(
     )
   end
 
+  # @param [String] area_code
+  # @param [String] country_code
+  # @param [String] phone_fingerprint
+  # @param [String] context the context of the OTP, either "authentication" for confirmed phones
+  # or "confirmation" for unconfirmed
+  # @param [String] otp_delivery_preference the channel used to send the message,
+  # either "sms" or "voice"
```

Comment:
we can also do this:
```suggestion
  # @param ["sms","voice"] otp_delivery_preference the channel used to send the message
```